pkg/wwan/mm-init.sh

```diff
@@ -23,7 +23,8 @@ enable_fcc_unlock() {
 }
 
 echo "Loading kernel modules used by ModemManager"
-modprobe -a qcserial usb_wwan qmi_wwan cdc_wdm cdc_mbim cdc_acm
+modprobe -a qcserial usb_wwan qmi_wwan cdc_wdm cdc_mbim cdc_acm \
+            wwan mhi mhi_pci_generic mhi_wwan_ctrl mhi_wwan_mbim
```

Comment:
Does the addition of these modules cause any significant size increase in the running kernel memory usage?

Reply:
I checked `/proc/modules` and `/proc/meminfo` between 12.0.0 and this PR, but do not see any significant kernel memory usage increase.

Reply:
Thanks